ci-operator/config/stolostron/submariner-addon/stolostron-submariner-addon-main.yaml

```diff
@@ -26,7 +26,6 @@ images:
       - registry.ci.openshift.org/stolostron/builder:go1.18-linux
   to: submariner-addon
 promotion:
-  disabled: true
```

Comment:
From @gurnben, promotion should be disabled on main.

Reply:
Ok, fixed.